src/EFCore.MySql/Migrations/MySqlMigrationsSqlGenerator.cs

```diff
@@ -391,16 +391,10 @@ protected override void Generate(DropSchemaOperation operation, IModel model, Mi
             builder
```

Comment:
The current implementation of [MigrationsSqlGenerator.Generate(CreateSequenceOperation operation, ...)](https://github.com/dotnet/efcore/blob/55e8f0a65e054ab6248474e4755d9e153c0f1c28/src/EFCore.Relational/Migrations/MigrationsSqlGenerator.cs#L521) has changed since this method was added (which we shouldn't have done in the first place) to `MySqlMigrationsSqlGenerator`. Therefore, take a look at the current base class implementation first.